test/test_item.rb

```diff
@@ -122,6 +122,11 @@ def test_get
 
     item = client.item.get(access_token)
     refute_empty(item.item)
+    refute_empty(item.status)
```

Comment:
`item.status` is `nil` in the tests. https://circleci.com/gh/plaid/plaid-ruby/241

Reply:
@be9 correct. Tests will fail until the API is updated. Right now the API does not return `status` for any ruby clients. Soon it will be updated to return status for any ruby client >= 6.0.0 and the tests here will pass and this assertion should be correct. Does that make sense? These tests were written preemptively :)